app/controllers/api/v1/rents_controller.rb

```diff
@@ -13,12 +13,15 @@ def create
         @rent = Rent.new(create_params_sanitized)
 
         if @rent.save
+          RentMailer.new_rent_notification(@rent.id).deliver_later
```

Comment:
I recently realize, the `@` here is useless here for define the object rent. Any doubt on why ask me.